client/my-sites/checkout/calypso-shopping-cart-provider.tsx

```diff
@@ -79,5 +79,11 @@ export default function CalypsoShoppingCartProvider( {
 
 function CalypsoShoppingCartMessages() {
 	const { responseCart, isLoading } = useShoppingCart();
-	return <CartMessages cart={ responseCart } isLoadingCart={ isLoading } />;
+	return (
+		<AsyncLoad
+			require="calypso/my-sites/checkout/cart/cart-messages"
```

Comment:
Do we also want to add `placeholder={ null }` or are we fine with having the placeholder there?

Reply:
Good thinking. This doesn't ultimately render anything; it's just used for its side effects.